woocommerce/woocommerce-ios
# [Store onboarding] Show onboarding tasks using `onboarding/tasks` endpoint.

Part of: #8892 

## Description
Starts showing the store onboarding tasks using the response from the `onboarding/tasks` endpoint. 

## Testing instructions
1. Launch the app and login if needed
1. In the "My Store" tab notice the store onboarding list.
1. The displayed tasks will be based on the status of your site. 
1. Tap on any of the tasks or the "View All" button to open the list in full screen.

Design for reference - VyLr7LvKodHE4kINfBE7Lw-fi-1644%3A82105

**Pending work**
- The scroll behaviour in the "My Store" is being worked on. - #9031 
- Handling the tap on each task in pending work. 

## Screenshots

https://user-images.githubusercontent.com/524475/222748898-0e02a8b5-6566-42ea-bc39-87d8d6015d90.mp4


---
